.github/workflows/export-optimized.yml

```diff
@@ -242,12 +251,20 @@ jobs:
         run: |
           cd godot
           source ../emsdk/emsdk_env.sh
-          scons p=web arch=wasm32 optimize=size lto=auto threads=no ${BUILD_OPTIONS}
+          scons p=web arch=wasm32 ${BUILD_OPTIONS} threads=no
+          mkdir -v -p ~/.local/share/godot/export_templates/${GODOT_VERSION}.${GODOT_RELEASE}/
+          mv ./bin/godot.web.template_release.wasm32.nothreads.zip ~/.local/share/godot/export_templates/${GODOT_VERSION}.${GODOT_RELEASE}/web_release.zip
```

Comment:
```suggestion
          mv ./bin/godot.web.template_release.wasm32.nothreads.zip ~/.local/share/godot/export_templates/${GODOT_VERSION}.${GODOT_RELEASE}/web_nothreads_release.zip
```

Reply:
also remove this wrong path https://github.com/MewPurPur/GodSVG/blob/main/export_presets.cfg#L386

Reply:
Thanks!